src/pages/MyBundlePage/MyBundle.style.ts

```diff
@@ -0,0 +1,42 @@
+import { css, styled } from 'styled-components';
+
+import { MOBILE } from '@/constants';
+
+export const Container = styled.article`
+  display: flex;
+  gap: 2rem;
+  padding: 4rem;
+  padding-top: 2rem;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  overflow-x: hidden;
+
+  @media screen and (max-width: ${MOBILE}px) {
+    height: auto;
```

Comment:
```suggestion
    height: auto;
    padding: 0.5rem;
```
P4; 모바일에서는 padding값이 조금 많이 줄어드는게 오히려 더 많은 텍스트를 보여줄 수 있을 것 같아요~!